.buildkite/hooks/pre-exit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+source .buildkite/scripts/tooling.sh
+
+set -euo pipefail
+
+unset_secrets
+
+# unset the environment variables not managged by "unset_secrets"
+if [[ "$BUILDKITE_PIPELINE_SLUG" == "elastic-package" && "$BUILDKITE_STEP_KEY" == "integration-parallel-gcp" ]]; then
```

Comment:
There would be any problem with unsetting all these variables always?
```suggestion
```

Reply:
mmm yes , and it probably it would be safer.
Executing those unset commands even if the variable does not exist should not cause any issue.
I'll change it.